lib/Dialect/FIRRTL/FIRRTLOpInterfaces.cpp

```diff
@@ -51,9 +51,14 @@ LogicalResult circt::firrtl::verifyModuleLikeOpInterface(FModuleLike module) {
     return module.emitOpError("requires valid port names");
   if (portNames.size() != numPorts)
     return module.emitOpError("requires ") << numPorts << " port names";
-  if (llvm::any_of(portNames.getValue(),
+  auto portNamesArray = portNames.getValue();
+  if (llvm::any_of(portNamesArray,
```

Comment:
Minor nit: these two loops could be merged.

Reply:
Good call, fixed thanks!